src/IO/GeoJSONWriter.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Brick\Geo\IO;
+
+use Brick\Geo\Exception\GeometryIOException;
+use Brick\Geo\Geometry;
+use Brick\Geo\GeometryCollection;
+
+/**
+ * Converter class from Geometry to GeoJSON.
+ */
+class GeoJSONWriter
+{
+    /**
+     * @param Geometry $geometry The geometry to export as GeoJSON.
+     *
+     * @return string The GeoJSON representation of the given geometry.
+     *
+     * @throws GeometryIOException If the given geometry cannot be exported as GeoJSON.
+     */
+    public function write(Geometry $geometry) : string
+    {
+        if ($geometry instanceof GeometryCollection
+            && $geometry->geometryType() === 'GeometryCollection') {
```

Comment:
Could you please add the following comment above this line, to explain why this line is necessary:

    // Filter out MultiPoint, MultiLineString and MultiPolygon